npm/ci-requirements.sh

```diff
@@ -14,21 +14,12 @@ pushd ./codegens/java-okhttp &>/dev/null;
   sudo rm -rf /var/lib/apt/lists/*
   sudo apt-get update
   sudo apt-get install -y openjdk-8-jdk
-  sudo wget http://central.maven.org/maven2/com/squareup/okhttp3/okhttp/3.9.1/okhttp-3.9.1.jar
-  sudo wget http://central.maven.org/maven2/com/squareup/okio/okio/1.14.0/okio-1.14.0.jar
+  unzip test/unit/fixtures/dependencies.zip
```

Comment:
Why are these changes showing up? 
These changes have already been committed.

Reply:
Actually I made a PR to my fork and done a rebase because my version of the fork was behind this commit. If you see the commit ID. it's the same as authored by you. So, It's not redundant but the same commit.

Reply:
it's not the same commit id. If it were, it wouldn't show up in the PR.

Reply:
Weird! Not much familiar with Github UI. Just noticed that. I just used GitHub to merge PR from existing repo to my fork and I chose to rebase while doing so. But I don't know how it happened.  No issues, I will delete this commit and force push it anyway. Thanks for informing this. It may have caused conflict merging already. I will rectify it. Anyway, it's WIP PR.